resources/charlie/README.md

```diff
@@ -0,0 +1,160 @@
+   Main PID: 826251 (casper-node)
+      Tasks: 26 (limit: 77016)
+     Memory: 34.9M
+     CGroup: /system.slice/casper-node.service
+             └─826251 /usr/bin/casper-node validator /etc/casper/config.toml
+
+Sep 10 10:37:15 joe-ubuntu casper-node[826251]: Sep 10 10:37:15.199 INFO  [casper_node::cli cli.rs:150] finished initialization
+Sep 10 10:37:15 joe-ubuntu casper-node[826251]: Sep 10 10:37:15.199 WARN  [casper_node::reactor reactor.rs:206] large event size, consider reducing it or boxing; event_size=184
+Sep 10 10:37:15 joe-ubuntu casper-node[826251]: Sep 10 10:37:15.202 INFO  [casper_node::components::small_network small_network.rs:166] 339c..c134: starting server background task; local_address=0.0.0.0:34553; public_address=127.0.0.1:34553
+Sep 10 10:37:15 joe-ubuntu casper-node[826251]: Sep 10 10:37:15.202 INFO  [casper_node::reactor reactor.rs:223] reactor main loop is ready
+Sep 10 10:37:15 joe-ubuntu casper-node[826251]: Sep 10 10:37:15.202 INFO  [casper_node::cli cli.rs:162] finished joining
+Sep 10 10:37:15 joe-ubuntu casper-node[826251]: Sep 10 10:37:15.202 INFO  [casper_node::components::small_network small_network.rs:694] 339c..c134: shutting down socket, no longer accepting incoming connections
+Sep 10 10:37:15 joe-ubuntu casper-node[826251]: Sep 10 10:37:15.202 WARN  [casper_node::reactor reactor.rs:206] large event size, consider reducing it or boxing; event_size=544
+Sep 10 10:37:15 joe-ubuntu casper-node[826251]: Sep 10 10:37:15.205 INFO  [casper_node::components::small_network small_network.rs:166] b076..384a: starting server background task; local_address=0.0.0.0:34553; public_address=127.0.0.1:34553
+Sep 10 10:37:15 joe-ubuntu casper-node[826251]: Sep 10 10:37:15.205 INFO  [casper_node::components::api_server api_server.rs:169] started HTTP server; addr=127.0.0.1:7777
+Sep 10 10:37:15 joe-ubuntu casper-node[826251]: Sep 10 10:37:15.205 INFO  [casper_node::reactor reactor.rs:223] reactor main loop is ready
+```
+
+### Reading logs
+
+The `journalctl` command is used to read logs from a systemd service. Yow specify the unit name and usually flags
+to jump to the end (-x) or follow (-f).
+
+This will show logs at the end of the logs:
+
+`journalctl -u casper-node -x`
+
+This will follow logs as they occur until stopped with Ctrl+C:
+
+`journalctl -u casper-node -f`
+
+### Starting and stopping services
+
+The service will start after installing a new .deb package, assuming `/etc/casper/config.toml` is in place.
+
+To start manually:
+
+`sudo systemctl start casper-node`
+
+To stop:
+
+`sudo systemctl stop casper-node`
+
+## Local Storage
+
+Unless specified in different location in `config.toml` the data for the node will
+be located in `/root/.local/share/casper-node` for a linux system.
+
+## Resetting and upgrading for new run
+
+Prior to a new upgrade and run, we need to clean up our local state.
+
+```
+sudo systemctl stop casper-node
+sudo apt remove casper-node
+sudo rm /root/.local/share/casper-node/*
```

Comment:
This command didn't work for me.  I had to do
```
sudo su -
rm /root/.local/share/casper-node/*
exit
```

Reply:
I'm suggesting `sudo rm -rf /root/.local/share/casper-node` as that seems to work and still be a one liner.